cocoa/src/toga_cocoa/hardware/camera.py

```diff
@@ -267,7 +267,10 @@ def __init__(self, interface):
             if self.interface.app.is_bundled:
                 raise RuntimeError(msg)
             else:
-                warnings.warn(msg)
+                warnings.warn(
+                    msg,
+                    stacklevel=2,
+                )
 
```

Comment:
If it *can* fit on one line, then it should; the separate args thing is only for when it can't.
```suggestion
                warnings.warn(msg, stacklevel=2)
```